modules/ossm-migrating-hub-how-to-use-migration-guides.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/migrating/ossm-migrating-from-2-to-3-assembly.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-migrating-hub-how-to-use-migration-guides_{context}"]
+== How to use the migration guides
```

Comment:
This heading would be a red flag for me if I was a user. It implies that the migration guide itself needs documentation for how to use it. I know that's not what you mean, though. Consider revising the heading to something like, "Migration guide overview".